contrib/win32/openssh/unittest-hostkeys.vcxproj

```diff
@@ -166,7 +166,7 @@
     </ClCompile>
     <Link>
       <SubSystem>Console</SubSystem>
-      <GenerateDebugInformation>No</GenerateDebugInformation>
+      <GenerateDebugInformation>true</GenerateDebugInformation>
```

Comment:
this can be "No"? 
We can set this to "true" on demand when we want to debug?
please take this as global comment as there are multiple locations where it can be fixed..